portal-frontend/src/app/features/notice-of-intents/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html

```diff
@@ -3,12 +3,12 @@ <h4>Submit Notice of Intent</h4>
 </div>
 
 <div mat-dialog-content class="content">
-  <p>Your notice of intent will be submitted to the {{ data.governmentName }}.</p>
+  <p>Your notice of intent will be submitted to the ALC.</p>
   <div>
     <div class="subheading2">Terms and Conditions:</div>
     <mat-checkbox #checkbox1 color="primary" class="checkbox">
-      I/we consent to the use of the information provided in the notice of intent and all supporting documents to process the
-      notice of intent in accordance with the Agricultural Land Commission Act, the Agricultural Land Reserve General
+      I/we consent to the use of the information provided in the application and all supporting documents to process the
```

Comment:
is this change intentional? This looks like NOI component

Reply:
Yes, this is according to the figma